src/SprykerEco/Zed/Unzer/UnzerConfig.php

```diff
@@ -434,4 +308,168 @@ public function getWebhookEventType(): string
     {
         return UnzerConstants::WEBHOOK_EVENT_TYPE;
     }
+
+    /**
+     * @api
+     *
+     * @return int
+     */
+    public function getExpensesRefundStrategyKey(): int
+    {
+        return $this->get(UnzerSharedConstants::EXPENSES_REFUND_STRATEGY_KEY);
+    }
+
+    /**
+     * @var array<int, string>
+     */
+    protected const UNZER_PAYMENT_STATE_OMS_STATUS_MAP = [
```

Comment:
constants should be placed above methods

Reply:
Fixed.